pkg/controller/constants/constants.go

```diff
@@ -39,4 +39,14 @@ const (
 
 	// ProvReqAnnotationPrefix is the prefix for annotations that should be pass to ProvisioningRequest as Parameters.
 	ProvReqAnnotationPrefix = "provreq.kueue.x-k8s.io/"
+
+	// WorkloadAnnotation is an annotation set on the Job's PodTemplate to
```

Comment:
So reading this I think its limiting to a K8s job. I think you mean a general kueue supported job but not sure what term we pick to reference this.

Reply:
Yes, this is supported for any Job (can be MPIJob, JobSet, etc.). In Kueue we say Job in many contexts meaning any of the CRDs, see the user-facing docs: https://kueue.sigs.k8s.io/docs/overview/.